lang/cs/Org.Apache.REEF.Common/Evaluator/IDriverConnection.cs

```diff
@@ -15,10 +15,12 @@
 // specific language governing permissions and limitations
 // under the License.
 
+using Org.Apache.REEF.Tang.Annotations;
 using Org.Apache.REEF.Utilities.Attributes;
 
 namespace Org.Apache.REEF.Common.Evaluator
 {
+    [DefaultImplementation(typeof(MissingDriverConnection))]
```

Comment:
It is Ok to do this way. If there is no default and the user also don't bind, Tang will also throw exception. In this way, we can control the error message though :)